src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -180,7 +180,7 @@ const useCloudEnvironmentPolling = googleProject => {
       const [newDisks, newRuntimes, galaxyDisks, cromwellDisks] = googleProject ? await Promise.all([
         Ajax(signal).Disks.list({ googleProject, creator: getUser().email }),
         Ajax(signal).Runtimes.list({ googleProject, creator: getUser().email }),
-        Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: tools.galaxy.appType }),
+        Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: tools.galaxy.appType, includeLabels: 'saturnWorkspaceName' }),
```

Comment:
Here also there will be  single call. I don't think the workspace name will ever be used though by WorkspaceContainer… why are you including the label here?